utils/qfirehose/Makefile

```diff
@@ -0,0 +1,58 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=qfirehose
+PKG_VERSION:=1.4.9
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/nippynetworks/qfirehose.git
+PKG_SOURCE_DATE:=2023-01-04
+PKG_SOURCE_VERSION:=f77d6729dcf72295e2c024782946cf8ac8a2d8c1
```

Comment:
What's wrong with using the 1.4.9 tag?

Reply:
Propably nothing (cannot see no with Mobile device on car if it is identical), when I flashed, I used git to acquire sources (master), so I built a package and leaned on solution to use same source code that I can verify that got the job done. Checking that out as soon as possible.